stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/message/MessageDao.kt

```diff
@@ -102,9 +107,10 @@ internal abstract class MessageDao {
     @Query("DELETE from stream_chat_message WHERE cid = :cid AND id = :messageId")
     abstract suspend fun deleteMessage(cid: String, messageId: String)
 
-    @Query("SELECT * FROM stream_chat_message WHERE stream_chat_message.id IN (:ids)")
     @Transaction
-    abstract suspend fun select(ids: List<String>): List<MessageEntity>
+    open suspend fun select(ids: List<String>): List<MessageEntity> {
+        return ids.mapNotNull { id -> select(id) }
```

Comment:
It will decrease a lot SQL Performance.
Why don't we chunk the list? :thinking: 
```
messagesIdList.chunked(950).flatMap { db.select(it) }
```